homeassistant/components/braviatv/__init__.py

```diff
@@ -64,31 +69,38 @@ class BraviaTVCoordinator(DataUpdateCoordinator[None]):
     several platforms.
     """
 
-    def __init__(self, hass, host, mac, pin, ignored_sources):
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        host: str,
+        mac: str,
+        pin: str,
+        ignored_sources: list[str],
+    ) -> None:
         """Initialize Bravia TV Client."""
 
         self.braviarc = BraviaRC(host, mac)
         self.pin = pin
         self.ignored_sources = ignored_sources
-        self.muted = False
-        self.channel_name = None
+        self.muted: bool | None = False
+        self.channel_name: str | None = None
         self.channel_number = None
-        self.media_title = None
-        self.source = None
-        self.source_list = []
-        self.original_content_list = []
-        self.content_mapping = {}
-        self.duration = None
-        self.content_uri = None
+        self.media_title: str | None = None
+        self.source: str | None = None
+        self.source_list: list[str] = []
+        self.original_content_list: list[str] = []
+        self.content_mapping: dict[str, str] = {}
+        self.duration: int | None = None
+        self.content_uri: str | None = None
         self.start_date_time = None
-        self.program_media_type = None
-        self.audio_output = None
-        self.min_volume = None
-        self.max_volume = None
-        self.volume_level = None
-        self.is_on = False
+        self.program_media_type: str | None = None
+        self.audio_output: str | None = None
+        self.min_volume: int | None = None
+        self.max_volume: int | None = None
+        self.volume_level: float | None = None
+        self.is_on: bool = False
```

Comment:
Here it'll be automatically inferred as `bool`. Manual type hint is not necessary.
```suggestion
        self.is_on = False
```